src/renderer/vanilla/util/html.js

```diff
@@ -61,18 +62,4 @@ const html = function (strings, ...values) {
 	return new TemplateLiteral(strings, ...values);
 };
 
-const unsafeHTML = (string) => {
-	return () => `${decodeAttribute(string)}`;
-};
-
-const spreadAttributes = (attributes) => {
-	return () => {
-		return Object.keys(attributes)
-			.map((key) => {
-				return `${camelToDash(key)}='${new Part(attributes[key])}'`;
-			})
-			.join(' ');
-	};
-};
-
-export { html, unsafeHTML, spreadAttributes };
+export { Part, TemplatePart, TemplateLiteral, html, unsafeHTML, spreadAttributes };
```

Comment:
unsafeHTML and spreadAttributes remain here for compatibility reasons? Probably fine.

Reply:
Yeah! I wasn't sure if it would break anything... Probably not. At least it shouldn't. But what do I know.